experimental/generation/generator/packages/library/test/forms/unittest_time.schema

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "TimeProperty": {
+            "type": "string",
+            "format": "time"
```

Comment:
Where is this file used?

Reply:
The array of unittestSchemaNames includes "time", the file is used for LU/LG/Dialog testing and Generated unit test schema .